apps/docs/app/api/assistant-ui-token/route.ts

```
@@ -0,0 +1,56 @@
+  }
+
+  cookieStore.set("jwt", getJwtForUser(userId), {
+    path: "/",
+    httpOnly: true,
+    secure: process.env.NODE_ENV === "production",
+    sameSite: "strict",
+    maxAge: 60 * 60 * 24 * 7, // 1 week
+  });
+
+  const client = new AssistantCloud({
+    apiKey: process.env["ASSISTANT_API_KEY"]!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)